README.md

```diff
@@ -93,26 +93,41 @@ Set your Ruby version with `rbenv`.
 ```bash
 brew install rbenv
 rbenv versions # see which ruby versions are already installed
-rbenv install 2.6 # install 2.6 if you don't have it already
-rbenv local 2.6 # rbenv local --unset reverses the action
+rbenv install  # the app currently uses version 2.6.10
+rbenv local # rbenv local --unset reverses the action
+
+# To switch between rbenv versions installed locally, use the following command:
+rbenv shell [version number]
+example: rbenv shell 2.6.10
 ```
 
 Set your Node version with `NVM`.
 
 ```bash
 brew install nvm
+brew info nvm # commands that shows the remaining steps to complete to install nvm properly
+mkdir ~/.nvm
 nvm install 14
 nvm use 14
+Add the following lines to your ~/.bashprofile or ~/.zshrc:
```

Comment:
Commenting the line makes it possible for someone to copy and paste the script  as a whole.

```suggestion
# Add the following lines to your ~/.bashprofile or ~/.zshrc:
```